readthedocs/api/v2/templates/restapi/footer.html

```diff
@@ -1,4 +1,6 @@
 <!-- Inserted RTD Footer -->
+{% load cache %}
+{% cache 900 footer %}
```

Comment:
This will cache the footer and use _the same footer for all the projects_. That's not correct. We need to add more keywords where.

Also, adding cache here, even if we use the proper variables/keywords, could cause confusions on users when a new version is activated, a translation is added, etc. So, we may want to invalidate the cache in those cases as well.